extensions/sonarcloud/vss-extension.json

```diff
@@ -53,6 +53,7 @@
     "type": "git",
     "uri": "https://github.com/SonarSource/sonar-scanner-vsts"
   },
+  "baseUri": "https://sonarcloud.io",
```

Comment:
Have you checked it doesn't break other resources (like icons or tasks)?

Reply:
I'll check again to be sure! Thanks!

Reply:
Seems to be all good!